.github/workflows/auto-update.yml

```diff
@@ -21,41 +16,71 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
         with:
-          fetch-depth: 0
+          token: ${{ secrets.KEPTN_BOT_TOKEN }}
+
+      - name: Checkout
```

Comment:
why two checkouts in a row?

Reply:
one is for the keptn.github.io repo and one is for the keptn/keptn repo
see: https://github.com/keptn/keptn.github.io/pull/1029/files#diff-fc41c8132b450c9cc5a4d687b8fad2774fbddc829b368b97888186542adf3d78R24